spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET /users' do
+    before do
+      FactoryBot.create :takashi
+      FactoryBot.create :satoshi
+    end
```

Comment:
以下のコードの方が、慣習的かつ、簡潔やね

Ref: https://qiita.com/kodai_0122/items/e755a128f1dade3f53c6

```suggestion
    before do
      create_list(:user, 2)
    end
```